bgpd/bgp_community.c

```diff
@@ -910,3 +910,132 @@ void community_finish(void)
 	hash_free(comhash);
 	comhash = NULL;
 }
+
+static unsigned int bgp_aggr_community_hash_key(void *data)
+{
+	struct community *community = data;
+
+	return jhash(community->val, community->size, 0xdcba4321);
+}
+
+static bool bgp_aggr_community_hash_cmp(const void *n1, const void *n2)
```

Comment:
isn't this community_cmp?

Reply:
I added a new function to safeguard the aggregation hash just in case the existing function goes through changes in the future.
I've made the changes to use the existing function.